install_config/install/advanced_install.adoc

```diff
@@ -512,6 +514,144 @@ xref:../../admin_guide/diagnostics_tool.adoc#additional-cluster-health-checks[Ad
 expiration can be found in
 xref:../../install_config/redeploying_certificates.adoc#install-config-redeploying-certificates[Redeploying Certificates].
 
+[[advanced-install-configuring-system-containers]]
+=== Configuring System Containers
+
+[IMPORTANT]
+====
+All system container components are
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+features in {product-title} 3.6. They must not be used in production and they
+are not supported for upgrades to {product-title} 3.6. During this Technology
+Preview phase, they are only meant for use with new cluster installations in
+non-production environments.
+====
+
+System containers provide a way to containerize services that need to run before
+the `docker` daemon is running. They are Docker-formatted containers that use:
+
+- link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/content_management_guide/managing_ostree_content[OSTree] for storage,
+- link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/#running_containers_with_runc[runC] for the runtime,
+- link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/#using_systemd_with_containers[systemd] for service management, and
+- link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/#using_skopeo_to_work_with_container_registries[skopeo] for searching.
+
+System containers are therefore stored and run outside of the traditional
+`docker` service. For more details on system container technology, see
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[Running System Containers] in the _Red Hat Enterprise Linux Atomic Host: Managing Containers_ documentation.
+
+You can configure your {product-title} installation to run certain components as
+system containers instead of their RPM or standard containerized methods.
+Currently, the `docker` and etcd components can be run as system containers in
+{product-title}.
+
+[WARNING]
+====
+System containers are currently OS-specific because they require specific
+versions of `atomic` and systemd. For example, different system containers are
+created for RHEL, Fedora, or CentOS. Ensure that the system containers you are
+using match the OS of the host they will run on.
+ifdef::openshift-enterprise[]
+{product-title} only supports RHEL and RHEL Atomic as the host OS, so by default
+system containers built for RHEL are used.
+endif::[]
+====
```

Comment:
@ashcrow Added the specific-OS version warning up here before the docker and etcd subsections as a general warning for system containers. If it's only applicable for container-engine, I can move it down into the docker subsection.